project.clj

```diff
@@ -104,10 +104,12 @@
                  ; chat protocols
                  [irclj "0.5.0-alpha4"]
                  ;; use this fork which uses javax.websockets for compatability with Yetibot
-                 [stylefruits/gniazdo-jsr356 "1.0.0"]
+                 ;; gniazdo 1.2.2 needed for discljord
+                 [stylefruits/gniazdo "1.2.2"]
```

Comment:
Needed to update this for discord to work properly. I'm thinking this will end up breaking other things since this uses `jetty`'s Websocket

Reply:
Yeah locally it is fine. I didn't test any other connectors though.

Reply:
I had to include this dep in the the `yetibot/yetibot`project.clj as well to get it to run properly.

Reply:
Confirmed slack still works with this upgrade.

Reply:
Sick. Thanks for checking that out.